python/tskit/tables.py

```diff
@@ -2873,10 +2881,10 @@ def __getstate__(self):
         return self.asdict()
 
     @classmethod
-    def load(cls, file_or_path):
+    def load(cls, file_or_path, skip_tables=False):
         file, local_file = util.convert_file_like_to_open_file(file_or_path, "rb")
         ll_tc = _tskit.TableCollection(1)
-        ll_tc.load(file)
+        ll_tc.load(file, skip_tables=bool(skip_tables))
```

Comment:
```suggestion
        ll_tc.load(file, skip_tables=skip_tables)
```

Reply:
Casting to bool here could have unintended consequences, as we might want to make ``skip_tables=None`` the default later (and infer a sensible default based on other param values). If users want to use Falsey behaviour they can cast to bool easily enough themselves.